src/main/java/com/gooddata/report/ReportService.java

```diff
@@ -7,129 +7,62 @@
 
 import com.gooddata.AbstractService;
 import com.gooddata.FutureResult;
-import com.gooddata.PollResult;
-import com.gooddata.GoodDataException;
-import com.gooddata.GoodDataRestException;
-import com.gooddata.SimplePollHandler;
-import com.gooddata.gdc.UriResponse;
+import com.gooddata.export.ExportService;
 import com.gooddata.md.report.Report;
 import com.gooddata.md.report.ReportDefinition;
-import com.fasterxml.jackson.databind.JsonNode;
-import com.fasterxml.jackson.databind.node.ObjectNode;
-import org.springframework.http.HttpEntity;
-import org.springframework.http.ResponseEntity;
-import org.springframework.http.client.ClientHttpResponse;
-import org.springframework.web.client.RestClientException;
 import org.springframework.web.client.RestTemplate;
 
-import java.io.IOException;
 import java.io.OutputStream;
 
+import static com.gooddata.report.ReportExportFormat.toExportFormat;
 import static com.gooddata.util.Validate.notNull;
-import static org.springframework.http.HttpMethod.GET;
-import static org.springframework.http.HttpMethod.POST;
 
 /**
  * Service for report export
+ * @deprecated use {@link com.gooddata.export.ExportService}
```

Comment:
What about to also use @Deprecated annotation? Compiler would probably not show warnings only based on javadoc.

Reply:
added annotations to all deprecated elements